devicetypes/smartthings/aeotec-doorbell-siren-6.src/aeotec-doorbell-siren-6.groovy

```diff
@@ -244,13 +244,13 @@ def resetActiveSound() {
 		sendHubCommand(onOffCmd(0x00))
 	}
 	sendEvent([name: "alarm", value: "off"])
-	sendEvent([name: "chime", value: "off"])
```

Comment:
Why did you remove this? The parent DTH also has the Chime capability.

You are also missing events for chime on lines 155 and 207.